core_backend/app/configs/llm_prompts.py

```diff
@@ -122,19 +122,17 @@ def get_supported_languages(cls) -> list[str]:
     """
 
 # ----  Question answering bot
-
-ANSWER_QUESTION_PROMPT = """
-    You are a high-performing question answering bot.
-    You support a maternal and child health chatbot.
-
-    Answer the user query in natural language by rewording the
-    following FAQ found below. Address the question directly and do not
-    respond with anything that is outside of the context of the given FAQ.
-
-    If the FAQ doesn't seem to answer the question, respond with
-    'Sorry, no relevant information found.'
-
-    Found FAQ: {faq}"""
+ANSWER_QUESTION_ERROR_MESSAGE = "Sorry, no relevant information found."
```

Comment:
We have the `STANDARD_FAILURE_MESSAGE` that we should use instead of this. We should change that to a globally accessible variable actually because it's defined twice in two different scripts. I commented on Sid's PR re. the fact that we should remove the second line from it.